sockets/docker/docker.go

```diff
@@ -115,10 +123,9 @@ func (p *Poller) pollCurrentConnectionsFor(container docker.APIContainers, file
 	if err != nil {
 		return err
 	}
-	var stdout, stderr bytes.Buffer
+	var stdout bytes.Buffer
```

Comment:
Hmm, where does stderr go? does it inherit from the parent process when it's not bound?